.pre-commit-config.yaml

```diff
@@ -8,4 +8,22 @@ repos:
         types: [file]
         files: \.(py|pyi)$
 
+-   repo: https://github.com/sqlalchemyorg/zimports
+    rev: 0.3.0
+    hooks:
+    -   id: zimports
+        types: [file]
+        files: \.(py|pyi)$
+
+-   repo: https://github.com/pycqa/flake8
```

Comment:
i had problems getting flake8 to work too, but if you know how to keep it working then great!

Reply:
Yeah, I made sure to get everything passing before I created the PR :)